src/utils/theme.ts

```diff
@@ -22,33 +22,33 @@ const DARK_HOVER_COLOR_CLASS: string = "dark:hover:text-dark-c";
 
 export {
     ACCENT_COLOR_CLASS,
-    TEXT_COLOR_CLASS,
-    HOVER_COLOR_CLASS,
-    DARK_TEXT_HOVER_CLASS,
     DARK_HOVER_COLOR_CLASS,
+    DARK_TEXT_HOVER_CLASS,
+    HOVER_COLOR_CLASS,
+    TEXT_COLOR_CLASS,
 };
 
 /**
  * Set the color scheme of the page
  */
 export function setColorScheme() {
-    // On page load or when changing themes, best to add inline in `head` to avoid FOUC
-    if (localStorage.theme === "dark") {
-        // Dark theme
-        document.documentElement.classList.add("dark");
-        Console.setDarkTheme();
-        visual?.theme(true);
-        Editor.setTheme(true);
-        GUI.setTheme(true);
-        darkModeToggle.innerHTML = "Dark Mode: On";
-    } else {
-        // Light theme
-        document.documentElement.classList.remove("dark");
-        Console.setLightTheme();
-        visual?.theme(false);
-        Editor.setTheme(false);
-        GUI.setTheme(false);
-        darkModeToggle.innerHTML = "Dark Mode: Off";
+    switch (localStorage.getItem("colorPreference")) {
+        case null:
```

Comment:
Should this be undefined instead?

Reply:
I've read it returns null if it is undefined. I actually haven't looked at documentation. Maybe we can test these things later today. I also thought that we might want to migrate users that set the theme to dark?!

Reply:
To my understanding of the difference between undefined and null, null would be expected, because undefined is its own type and null is just the undefined value of a type. So if I used `localStorage.colorPreference`, I would assume to get undefined returned because the variable itself is even missing. Right?